.github/workflows/deploy-on-release.yml

```diff
@@ -51,6 +49,3 @@ jobs:
         run: |
           aws ecs update-service --cluster wca-on-rails --service wca-on-rails-prod-auxiliary-services --force-new-deployment
         # There is no pipeline for staging so we manually force to update the image
```

Comment:
If you're removing the step then you should probably also remove the comment ;)

Reply:
oops done